util/src/main/java/io/kubernetes/client/informer/cache/ProcessorListener.java

```diff
@@ -33,7 +30,7 @@ public ProcessorListener(ResourceEventHandler<ApiType> handler, long resyncPerio
     this.resyncPeriod = resyncPeriod;
     this.handler = handler;
 
-    this.queue = new ArrayBlockingQueue<>(DEFAULT_QUEUE_CAPACITY);
+    this.queue = new LinkedBlockingQueue<>();
```

Comment:
this is interesting.. i roughly searched and understood the difference between between `ArrayBlockingQueue` and `LinkedBlockingQueue`. and the latter indeed wins in the unbounded case.. is there a third option for blocking queue?

Reply:
I have searched implementations of BlockingQueue in our current libraries and the LinkedBlockingQueue is most suitable for our case. Do you have any suggestion for it?